oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -109,9 +110,9 @@
   private final Collection<String> defaultScopes;
   private final String quotaProjectId;
   private final int lifetime;
+  private final boolean useJWTAccessWithScope;
```

Comment:
nit: this should be useJwtAccessWithScope. Sorry, I misstyped my suggestion :) per Jon, if used within name, JWT should be Jwt.

Reply:
yea, updated the name.